### PR TITLE
[v1.5][F11-B3] JobbiBot

### PR DESCRIPTION
**Pull request by ghost**
Updated documentations @niloc94 @TanCiKang

### CODE REVIEW

**Comment by ghost** [comment]
v1.1 is added here too

File: README.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
-= Address Book (Level 4)
```
Comment (OuyangDanwen) [inline README.adoc:1]: The introduction below needs to be changed accordingly. This is no longer an address book application.

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -3,22 +3,23 @@
...
 _{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
 {empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore]
+and https://www.yale-nus.edu.sg[Yale-NUS College].
 
 == Project Team
 
-=== John Doe
+=== Wyin Kok
```
Comment (OuyangDanwen) [inline docs/AboutUs.adoc]: Remember to update the pictures and roles by next tutorial.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -840,10 +853,103 @@ Use case ends.
...
+=== Use case: Find Internship
+
+*System*: Chat Bot
+
+*MSS*
+
+1. User keys in command to find internship
+2. Chat Bot gives user a list of prompts ( e.g. Salary, duration )
+3. User follows the prompts and key in the relevant details
+4. Chat Bot replies with a list of internship companies
```
Comment (OuyangDanwen) [inline docs/DeveloperGuide.adoc]: Please follow the forth-back interaction. 3 and 4 should not be both from the chat bot. It is a format issue.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -840,10 +853,103 @@ Use case ends.
...
+=== Use case: Explore Internship
+
+*System*: Chat Bot
+
+*MSS*
+
+1. User keys in command to explore internship
+2. Chat Bot gives user a list of prompts ( common skills sets )
+3. User fills in a prompt or a list of prompts
+4. Chat Bot replies with a list of internship companies
```
Comment (OuyangDanwen) [inline docs/DeveloperGuide.adoc]: Same here.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -100,13 +100,12 @@ Edits the name of the 2nd person to be `Betsy Crower` and clears all existing ta
 
 === Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords. +
+Finds persons whose names, phone, email or address contain any of the given keywords. +
```
Comment (OuyangDanwen) [inline docs/UserGuide.adoc]: For UG, the team should include ALL the features you are going to implement. By now, you should know what they are. Mark them accordingly based on whether you think it as a V1.5, V2.0 feature. The current version is far from complete.

**Review by OuyangDanwen — COMMENTED** [review]
A few things: 1. Follow the MSS format for interactions. 2. Update user guide as soon as possible. Based on the requirements, you should update all features by last tutorial. The task is NOT update the features along the way (This is missed and as a result, I could not give you the team component mark). 3. Update all necessary admin info (photos, etc). 4. Change the relevant parts to correspond to your project rather than AB4 (product name, etc). 5. Update PR title to reflect the current version. Please take not of the above points. @wyinkok @TanCiKang @niloc94 Please refer to F11 B4 to see their changes. They also missed the UG changes, but the rest of the docs were well updated.

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -10,7 +10,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4/tree/master
+:repoURL: https://github.com/CS2103JAN2018-F11-B3/tree/master
```
Comment (OuyangDanwen) [inline docs/DeveloperGuide.adoc:13]: Title of the DG should be changed. Not AB4.

**Review by OuyangDanwen — COMMENTED** [review]
Changes made are generally okay. Note that for DG, try to add in diagrams to explain your new features. It is also good to have for your portfolio. Meanwhile, for your diagrams, eg. ModelDesign1.PNG, ModelDesign2.PNG, draw the arrows, joining and splitting of lines properly. ModelDesign2.PNG has wrong splitting and joining for RequiredInformation.  Also think about the creation of classes. Do you need a class to hold every single piece of information, even if it is just a single string? Lastly, make sure the UG is ready before the peer testing.

**Pull request closed without merging**